workflows/diagnostics/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -103,7 +103,9 @@ class RunDiagnostics:
     def __post_init__(self):
         # indexes for faster lookup
         self._attrs = {ds.run: ds.attrs for ds in self.diagnostics}
-        self._varnames = {ds.run: set(ds) for ds in self.diagnostics}
+        self._varnames = {
+            ds.run: set(ds).union(set(ds.coords)) for ds in self.diagnostics
```

Comment:
This is needed so that I can access the time coordinate to infer the length of a sample run within the `infer_duration_seconds` function.  Otherwise this Dataset-wrapper only recognizes data variables.  A slightly cleaner way to obtain the names of all the variables (data variables and coordinates) in a Dataset is to use `set(ds.variables)`, so I've switched to that.

If you think there is a cleaner way to obtain the time coordinate in this context I'm open to learning about it, however.

Reply:
Ah okay, that sounds good.